Makefile

```diff
@@ -14,133 +14,79 @@ DOCKER_IMAGE_NAME ?= cert-manager/sample-external-issuer/controller
 # Image URL to use all building/pushing image targets
 IMG ?= ${DOCKER_REGISTRY}/${DOCKER_IMAGE_NAME}:${VERSION}
 
-# BIN is the directory where tools will be installed
-export BIN ?= ${CURDIR}/bin
-
-OS := $(shell go env GOOS)
-ARCH := $(shell go env GOARCH)
-
-# Kind
-KIND_VERSION := 0.18.0
-KIND := ${BIN}/kind-${KIND_VERSION}
-K8S_CLUSTER_NAME := sample-external-issuer-e2e
 
 # cert-manager
 CERT_MANAGER_VERSION ?= 1.11.1
 
-# Controller tools
-CONTROLLER_GEN_VERSION := 0.11.3
-CONTROLLER_GEN := ${BIN}/controller-gen
```

Comment:
Moved tool versions and download variables to the Kubebuilder "Build dependencies" section at the bottom.